kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/create-domain-job-template.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
 spec:
   template:
     metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
```

Comment:
Are we adding this annotation regardless of whether istio_enabled is true or false? Is it better not to add it if istio_enabled is false?